ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -2296,3 +2300,40 @@ def test065_download_suborder_sheet_for_single_order(self):
                                   f"{str(fixed_sheet_row_data)} : {str(formatted_orders[index])}")
             for item in formatted_orders[index]:
                 self.assertIn(item, fixed_sheet_row_data)
+
+    @parameterized.expand(['sub_script', 'super_script'])
+    def test068_filter_testunit_by_scripts(self, value):
+        """  Make sure that user can filter by sub & super scripts in the filter drop down list
+             LIMS-7447
+        """
+
+        self.test_unit_api.set_name_configuration()
+        if value == 'sub_script':
+            api, testunit = self.test_unit_api.create_qualitative_testunit(unit='14[158]')
+            self.assertEqual(api['status'], 1)
+        else:
+            api, testunit = self.test_unit_api.create_qualitative_testunit(unit='15{158}')
+            self.assertEqual(api['status'], 1)
+
+        unit = testunit['unit']
+        self.orders_page.get_orders_page()
```

Comment:
why you need to navigate to order page, while set up gets order page and you didn't navigated to another page in previous steps